src/emulator/relocation.cpp

```diff
@@ -192,7 +193,7 @@ static bool relocate(void *data, Code code, uint32_t s, uint32_t a, uint32_t p)
         return true;
     }
 
-    std::cerr << "Unhandled relocation code " << code << "." << std::endl;
+    LOG_ERROR("Unhandled relocation code {}.", code);
```

Comment:
This should actually be a warning. Some homebrew has unhandled codes and they seem to be OK to ignore.